lib/streamlit/dataframe_util.py

```diff
@@ -783,12 +949,24 @@ def _unify_missing_values(df: DataFrame) -> DataFrame:
     return df.fillna(np.nan).replace([np.nan], [None])
 
 
+def _pandas_df_to_series(df: DataFrame) -> Series[Any]:
+    """Convert a Pandas DataFrame to a Pandas Series by selecting the first column."""
```

Comment:
nit: I would add the info that it raises an exception when the dataframe has more than 1 columns.

Reply:
Added 👍